src/Stepping.TmProviders.LocalTm/Stepping/TmProviders/LocalTm/HostedService/LocalTmHostedService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Stepping.TmProviders.LocalTm.DistributedLocks;
+using Stepping.TmProviders.LocalTm.TransactionManagers;
+
+namespace Stepping.TmProviders.LocalTm.HostedService;
+
+public class LocalTmHostedService : IHostedService
```

Comment:
What do you think about the name `HostedServiceLocalTmProcessor` and an implemented interface `ILocalTmProcessor`?